app/controllers/submissions_controller.rb

```diff
@@ -218,7 +218,9 @@ def download
 
     elsif params[:annotated]
       @problems = @assessment.problems.to_a
```

Comment:
tbh, this seems like dead code that can be removed altogether. To be sure, try making a pdf submission, annotate it, then download it.

Reply:
I think I use the @problems declaration to retrieve the problems and sort them. Let me know if you think it's dead code and I'll test it out though!